src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -301,8 +301,16 @@ private Object parseValue(StaplerRequest request) throws ServletException, IOExc
             }
         }
 
-        // TODO: perhaps we should return a different object to allow the workflow to look up
-        // who approved it, etc?
+        // If requested, add the approver to the list of returned parameters.
+        String submitter = input.getSubmitter();
+        if (submitter != null) {
+            Authentication a = Jenkins.getAuthentication();
```

Comment:
Could be moved inside the `if`-block, since the value is used iff `valueName != null`.